tests/test_install.sh

```diff
@@ -45,43 +47,19 @@ if [[ "$1" == "install" ]]; then
   if [[ "${REPLY}" == 'y' ]]; then
     echo "########################################################################################"
     echo "#  Waiting for NGINX Ingress Controller to start"
-    echo "########################################################################################"      
-
-    helm upgrade --install ingress-nginx ingress-nginx \
-    --repo https://kubernetes.github.io/ingress-nginx \
+    echo "########################################################################################" 
+    helm repo add ${INGRESS_HELM_CHART} ${INGRESS_HELM_REPO}
+    helm install ${INGRESS_HELM_CHART} ${INGRESS_HELM_CHART}/ingress-nginx \
     --namespace ${INGRESS_NAMESPACE} --create-namespace \
-    --set controller.updateStrategy.rollingUpdate.maxUnavailable=1 \
-    --set controller.hostPort.enabled=true \
-    --wait --timeout=60s
-  else
-    echo "########################################################################################"
-    echo "#  Skipping NGINX Ingress Controller installation"
-    echo "########################################################################################"
-  fi
-
-  echo "########################################################################################"
-  echo "#  Install OWASP Juice Shop Application"
-  echo "#  "
-  echo "#  Documentation: https://artifacthub.io/packages/helm/securecodebox/juice-shop"
-  echo "########################################################################################"
-  read -r -p 'Install OWASP Juice Shop [y/n]: '
-
-  if [[ "${REPLY}" == 'y' ]]; then
-    helm repo add securecodebox https://charts.securecodebox.io/
+    --wait --timeout=90s
 
-    echo "########################################################################################"
-    echo "#  Waiting for Juice Shop to start"
-    echo "########################################################################################"      
+    # --set controller.updateStrategy.rollingUpdate.maxUnavailable=1 \
+    # --set controller.hostPort.enabled=true \
+    # --set controller.service.loadBalancerIP=127.0.0.1 \
```

Comment:
Why are these commented out?

Reply:
Felipe was having issues on his Manjao Linux host getting the LoadBalancer to work.  It was pending on the EXTERNAL_IP when you show services.  Can we leave until we sort that out?
Also hostPort causes port conflicts when you want to run multiple workers on the same node.  I've tested the standalone with 5 workers.  Since I only have on node it was a problem.  I'd assume in prod we'd have one worker per node?

Reply:
This is now addressed with the DEVELOP/PROD settings.